e2e/e2etests/test_bitcoin_withdraw_invalid.go

```diff
@@ -0,0 +1,46 @@
+package e2etests
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/btcsuite/btcutil"
+	"github.com/zeta-chain/zetacore/e2e/runner"
+	"github.com/zeta-chain/zetacore/e2e/utils"
+)
+
+func TestBitcoinWithdrawToInvalidAddress(r *runner.E2ERunner) {
+	// try to withdraw 0.00001 BTC from ZRC20 to BTC legacy address
+	// first, approve the ZRC20 contract to spend 1 BTC from the deployer address
+	WithdrawToInvalidAddress(r)
+}
+
+func WithdrawToInvalidAddress(r *runner.E2ERunner) {
+
+	amount := big.NewInt(0.00001 * btcutil.SatoshiPerBitcoin)
+
+	// approve the ZRC20 contract to spend 1 BTC from the deployer address
+	tx, err := r.BTCZRC20.Approve(r.ZevmAuth, r.BTCZRC20Addr, big.NewInt(amount.Int64()*2)) // approve more to cover withdraw fee
```

Comment:
Might not be enough and might be the cause of the current error. I think we can set a hardcoded high number for the approval